pkgs/development/tools/kexpand/default.nix

```
@@ -0,0 +1,18 @@
+{ buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "kexpand-2017-05-12";
```

```suggestion
  name = "kexpand-unstable-2017-05-12";
```
https://nixos.org/nixpkgs/manual/#sec-package-naming